### ISSUE TITLE
Support to directly pass the certificate as string/binary to Relay Proxy

### ISSUE THREAD

**Issue by ujayant** [launchdarkly/python-server-sdk#266]
**Is your feature request related to a problem? Please describe.**
There is no support to pass the certificate as a string directly while using the Relay Proxy. We can only pass the certificate file path. In Go library, though, we can directly pass the certificate key or use the file path option while using the Relay Proxy.

[Description in the code which requires us to pass the path to the certificate file](https://github.com/launchdarkly/python-server-sdk/blob/main/ldclient/config.py#L104)

[Method from the Go SDK which allows us to pass the certificate as a bytes array](https://github.com/launchdarkly/go-server-sdk/blob/v7/ldcomponents/http_configuration_builder.go#L71)

**Describe the solution you'd like**
Is there a plan to add the support to pass the certificate as a string/binary date while using the Relay Proxy in Python SDK? It looks like this support can be added by upgrading the `urllib3` library version to `>=2.0.0`.

**Describe alternatives you've considered**

**Additional context**
Add any other context about the feature request here.

**Comment by keelerm84** [comment]
Thank you for bringing this issue to our attention.

I do think this is something we can support in the next major release (since the version change constitutes a breaking change). However, I cannot give a timeline on when that might be. 

I have filed this request in our internal tracker as sc-232759. I will update you on this issue once we have added the requested support.

Again, thank you for letting us know about this request!

**Comment by ujayant** [comment]
@keelerm84 Do you have any further updates on this request?

**Comment by keelerm84** [comment]
No, I don't have an update on this yet. As I had mentioned before, this would require a breaking change. We strive to limit those unless absolutely necessary, and typically only when the team has available capacity to tackle some of the other larger concerns.

However, I will work to make sure this is done during that next change cycle.
